Modules/Remote/BSplineGradient.cmake

```diff
@@ -0,0 +1,5 @@
+itk_fetch_module(BSplineGradient
+  "Approximate an image's gradient from a b-spline fit to its intensity."
+  GIT_REPOSITORY ${git_protocol}://github.com/dzenanz/ITKBSplineGradient.git
```

Comment:
This should point to

  https://github.com/InsightSoftwareConsortium/ITKBSplineGradient

Reply:
Done.